src/Proto.Actor/EventStream.cs

```diff
@@ -56,9 +56,18 @@ public Subscription<T> Subscribe(Action<T> action)
 
         public void Publish(T msg)
         {
+            var logger = Log.CreateLogger<EventStream>();

```

Comment:
Loggers shouldn't be inherited.  They're unique to the object and type.  Honestly, there shouldn't be a base class with a logger in them at all.